sklearn/naive_bayes.py

```diff
@@ -512,8 +512,37 @@ class _BaseDiscreteNB(_BaseNB):
 
     __init__
     _joint_log_likelihood(X) as per _BaseNB
+    _update_feature_log_prob(alpha)
+    _count(X, Y)
     """
 
+    @abstractmethod
+    def _count(self, X, Y):
+        """Update counts that are used to calculate probabilities.
+
+        The counts make up a sufficient statistic extracted from the data.
+        Accordingly, this method is called each time ``fit`` or ``partial_fit``
+        update the model. The number and composition of counts depend on a
+        concrete model, but ``self.class_count`` must be updated in any case.
+
+        Parameters
+        ----------
+        X : array-like of shape (n_samples, n_features)
+            The input samples.
+        Y : array-like of shape (n_samples, n_classes)
```

Comment:
```suggestion
        Y : ndarray of shape (n_samples, n_classes)
```

Reply:
@jeremiedbb Thank you for corrections and explanations. I committed them all.